extensions-contrib/opentelemetry-emitter/README.md

```diff
@@ -0,0 +1,118 @@
+# OpenTracing Emitter
+
+The OpenTelemetry emitter generates OpenTelemetry Spans for queries.
+
+
+## Configuration
+
+### Enabling
+
+Load the plugin and enable the emitter in `common.runtime.properties`.
+
+Load the plugin:
+
+```
+druid.extensions.loadList=[..., "opentelemetry-emitter"]
+```
+
+Enable the emitter:
+
+```
+druid.emitter=opentelemetry
```

Comment:
since the opentelemetry emitter doesn't support metrics, it's unlikely someone would only enable that emitter. Maybe the example should be more realistic and show how to add it to a config with an existing emitter.